extension/popup/popupView.jsx

```diff
@@ -480,8 +480,18 @@ const VoiceInput = ({ suggestions, onClickLexicon }) => {
   };
   return (
     <div id="voice-input">
-      {suggestions ? (
+      {buildSettings.inDevelopment ? (
         <div id="suggestions">
+          <p>Developer tip:</p>
+          <a
+            target="_blank"
+            rel="noopener"
+            id="lexicon"
+            href="../views/lexicon.html"
+            onClick={onInputStarted}
+          >
+            Start typing to make a request using a keyboard
+          </a>
```

Comment:
Instead put a `<div>` around this new message, and put the conditional around that, like `{buildSettings.inDevelopment ? <div><p>Developer tip:...</div> : null}`